charts/redpanda/Chart.yaml

```
@@ -23,7 +23,7 @@ type: application
 # The chart version and the app version are not the same and will not track
 # together. The chart version is a semver representation of changes to this
 # chart.
-version: 5.7.41
+version: 5.7.42
```

If the first commit replaces commonLabels with new value, then I would suggest the following bump
```suggestion
version: 5.8.1
```